spec/csv_row_model/validators/date_time_format_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe DateTimeFormatValidator do
+  let(:klass) do
+    Class.new do
+      include ActiveWarnings
+      attr_accessor :string1
+      warnings do
+        validates :string1, date_format: true
```

Comment:
To be honest I don't know where `date_format` comes from, it was a copy past on `date_format_spec.rb` I guess is a attribute of `ActiveModel::Validations`